pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -108,6 +108,7 @@ type KubernetesMetaConfiguration struct {
 	EnableReadinessProbe                     bool                `json:"enable_readiness_probe,omitempty"`
 	EnableCrossNamespaceSecret               bool                `json:"enable_cross_namespace_secret,omitempty"`
 	EnableFinalizers                         *bool               `json:"enable_finalizers,omitempty"`
+	EnablePostgresTopologySpreadConstraints  bool                `json:"enable_postgres_topology_spread_constraints,omitempty"`
```

Comment:
We don't need this global option here. Either one specifies topology spread constraints in the manifest or not.
Adding options would also require changes in many more parts of the code, which the PR currently lacks. So just drop the line.